Week1/Day3/ExercisesXPNinja/code.py

```diff
@@ -0,0 +1,41 @@
+companies = "Volkswagen, Toyota, Ford Motor, Honda, Chevrolet"
+companies_list = companies.split(",")
+print(f"{len(companies_list)} companies in the list")
+companies_list.sort(reverse=True)
+print(companies_list)
+
+companies_with_o = []
+for company in companies_list:
+    if 'o' in company.lower():
+        companies_with_o.append(company)
+
+print("Number of companies names with letter ‘o’:", len(companies_with_o))
+
+companies_without_i = []
+for company in companies_list:
```

Comment:
any idea how to do it with one line?

Reply:
companies_without_i = [company for company in companies_list if 'i' not in company]